paasta_tools/cli/cmds/status.py

```diff
@@ -790,42 +786,11 @@ def _print_flink_status_from_job_manager(
         output.append(PaastaColors.red("    Flink cluster is not available yet"))
         return 1
 
-    # Print Flink Config SHA
-    # Since metadata should be available no matter the state, we show it first. If this errors out
-    # then we cannot really do much to recover, because cluster is not in usable state anyway
     metadata = flink.get("metadata")
-    labels = metadata.get("labels")
-    config_sha = labels.get(paasta_prefixed("config_sha"))
-    if config_sha is None:
-        raise ValueError(f"expected config sha on Flink, but received {metadata}")
-    if config_sha.startswith("config"):
-        config_sha = config_sha[6:]
-    output.append(f"    Config SHA: {config_sha}")
+    ecosystem = system_paasta_config.get_ecosystem_for_cluster(cluster)
 
-    if verbose:
-        # Print Flink repo links
-        output.append(f"    Repo(git): https://github.yelpcorp.com/services/{service}")
-        output.append(
-            f"    Repo(sourcegraph): https://sourcegraph.yelpcorp.com/services/{service}"
-        )
-
-        # Print Flink Pool information
-        flink_pool = flink_instance_config.get_pool()
-        output.append(f"    Flink Pool: {flink_pool}")
-
-        # Print ownership information
-        flink_monitoring_team = flink_instance_config.get_team() or get_team(
-            overrides={}, service=service, soa_dir=DEFAULT_SOA_DIR
-        )
-        output.append(f"    Owner: {flink_monitoring_team}")
-
-        # Print rb information
-        flink_rb_for_instance = flink_instance_config.get_runbook() or get_runbook(
-            overrides={}, service=service, soa_dir=DEFAULT_SOA_DIR
-        )
-        output.append(f"    Flink Runbook: {flink_rb_for_instance}")
-
-    # Print Flink Version
+    # Get flink config if running
```

Comment:
imo, these `Get * if running` comments would be more valuable if they explained why there's a special case here

e.g.,
```
A flink cluster can be in $STATES, but we can only provide additional data for clusters in a running state
```